monsteer/CMakeLists.txt

```diff
@@ -18,5 +18,5 @@ set(MONSTEER_LINK_LIBRARIES PUBLIC Brion Lunchbox zeq)
 
 common_library(Monsteer)
 
+add_subdirectory(brionplugin)
```

Comment:
I find "plugin" alone not sufficiently informative. I know our views differ on this, but for me clarity is more important than brevity, and "brionplugin" is good because when you open the project you immediately see what is there without having to guess.
The second idea of reversing the directories is very interesting because it aligns better with the actual name of the library (BrionMonsteerSpikeReport). I'm not sure if it is a common practice to use the namespace of a different project, but I think it can be a valid alternative.